server/routes/bucketlist-routes.js

```diff
@@ -6,5 +6,5 @@ const router = express.Router();
 router.post('/', bucketlistController.createBucketList);
 router.get('/', bucketlistController.getBucketlists);
 router.put('/:bucketlistId', bucketlistController.updateBucketList);
```

Comment:
URLs should not have any uppercase characters. you can have `bucketlist-id`.

This can be overlooked on the backend but on the front-end, no uppercase. 

you could also leave this as `/:id` in this context